animareflection/ui
# Toast Component

## Description

##### Task link: https://trello.com/c/dB87gRHB/169-toast-component

Added custom `Toast` and `Toaster` components.

## Test Steps

1) Verify that structure is sound for component(s), recipe, stories, and test.
2) Verify that all tests pass.
3) Verify that `Toast`s render and function appropriately in storybook and downstream apps.
4) Verify that styles are appropriate.
